### ISSUE TITLE
Replace Simpsons seed data with faker

### ISSUE THREAD

**Issue by revjtanton** [CodeForBaltimore/Bmore-Responsive#246]
### Task
Instead of the JSON let's grab a ton of fake data and really fill out a test build!

### Acceptance Criteria
- [ ] faker is used to seed data
- [ ] Simpsons JSON is removed.

### Notes
https://www.npmjs.com/package/faker

**Comment by blakenan-bellese** [comment]
@revjtanton would this still be invoked by the `npm run db-seed` command?

**Comment by blakenan-bellese** [comment]
@revjtanton I'm inclined to suggest that this faker include at least one facility of each of the types that are useful in HCRC (Assisted Living Facility, Mixed Housing, Senior Housing), but recognize that this would create a logical dependency between the two projects.  Any suggestion on how to avoid?  Can we somehow have an HCRC-faker, maybe as part of the HCRC project?

**Comment by mschechter-bellese** [comment]
@blakenan-bellese - I think that an HCRC faker would be good for populating an instance of Bmore-Responsive to use for HCRC development (maybe a separate project?); Bmore-Responsive has no functionality keyed off entity types, and we can test all the functionality around entities without having that data be specific to a use case such as HCRC.

Could you please clarify whether you want to use the faker to generate the JSON used for seeding, or use it dynamically during the seeding process? The latter is much more complex, and would also add test complexity because we would not have any known data we could reference explicitly.

**Comment by blakenan-bellese** [comment]
I agree with your comments that an “HCRC Faker” may not belong in the Bmore Responsive repo.  The motivation was definitely to make the test data set more aligned with HCRC data needs.  I’m indifferent as to the faker is used, therefore,  easier is always better.  Simply generating JSON that can be used for seeding would be sufficient.


[signature_843658801]

Bill Lakenan
Bellese Technologies | bellese.io
***@***.******@***.***>  |  443-914-2126
300 Red Brook Blvd, Owings Mills, MD 21117
[https://lh3.googleusercontent.com/-l-MSfbYFAnCvqklJE30UrLFPpNrtvIWKu3Y-7b8YfHhjFo5eLZjcMtoctL7cl6EWWc1MBBdxdfmf0w2fXGodEWk01bd6RtrvBC1LqjAYbvItinODkiEPVjeIYKSHJap3qwIxjsM]<https://www.linkedin.com/company/2705760/> [https://lh4.googleusercontent.com/QNJbiLeSHzDK4DPi0QCoSi7jcoWfVnDUMUPCVXm86rk_4iRu8RasvhalLLiXNE7EqHEp8ARRWItZQ8PSIB4Np7HFC520sEBYwbUCE4as5ykVcog22WWDeJQesZDEAd14u3MubYPs] <https://www.facebook.com/Bellese-Technologies-490279364472269>


From: Michael Schechter ***@***.***>
Reply-To: CodeForBaltimore/Bmore-Responsive ***@***.***>
Date: Thursday, April 15, 2021 at 11:36 AM
To: CodeForBaltimore/Bmore-Responsive ***@***.***>
Cc: Bill Lakenan ***@***.***>, Mention ***@***.***>
Subject: Re: [CodeForBaltimore/Bmore-Responsive] Replace Simpsons seed data with faker (#246)


@blakenan-bellese<https://github.com/blakenan-bellese> - I think that an HCRC faker would be good for populating an instance of Bmore-Responsive to use for HCRC development (maybe a separate project?); Bmore-Responsive has no functionality keyed off entity types, and we can test all the functionality around entities without having that data be specific to a use case such as HCRC.

Could you please clarify whether you want to use the faker to generate the JSON used for seeding, or use it dynamically during the seeding process? The latter is much more complex, and would also add test complexity because we would not have any known data we could reference explicitly.
